examples/tutorials/basics/text.py

```diff
@@ -12,146 +12,233 @@
 import pygmt
 
 # %%
-# Basic map annotation
-# --------------------
+# Adding a single text label
+# --------------------------
 #
-# Text annotations can be added to a map using the :meth:`pygmt.Figure.text`
-# method of the :class:`pygmt.Figure` class.
-#
-# Here we create a simple map and add an annotation using the ``text``, ``x``,
-# and ``y`` parameters to specify the annotation text and position in the
-# projection frame. ``text`` accepts *str* types, while ``x`` and ``y``
-# accept either *int* or *float* numbers, or a list/array of numbers.
+# To add a single text label to a plot the ``text`` and ``x`` and ``y`` parameters to
+# specify the text and position.
```

Comment:
This sentence doesn't read well. Missing a verb?
```suggestion
# To add a single text label to a plot, use the ``text`` and ``x`` and ``y`` parameters to
# specify the text and position.
```

Reply:
Oh, yes, thanks for spotting!